Sources/Toast.swift

```diff
@@ -129,7 +129,8 @@ open class Toast: Operation {
     DispatchQueue.main.async {
       self.view.setNeedsLayout()
       self.view.alpha = 0
-      ToastWindow.shared.addSubview(self.view)
+      UIApplication.shared.windows[UIApplication.shared.windows.count - 1].addSubview(self.view)
```

Comment:
In Swift, to avoid race conditions if any between the two accesses to `UIApplication.shared.windows`, it would be safer to write the same as:
```
UIApplication.shared.windows.last?.addSubview(self.view)
```

(note: this is a syntax comment, it doesn't mean that I approve replacing `ToastWindow.shared` with it)

Reply:
Getting window like this can occur some bugs. We can't expect which window will be the last.